pkg/agent/route/interfaces.go

```diff
@@ -38,6 +38,16 @@ type Interface interface {
 	// It should do nothing if the routes don't exist, without error.
 	DeleteRoutes(podCIDR *net.IPNet) error
 
+	// AddVirtualIPToAntreaGatewayRoutes adds routes from virtual IP to antrea-gw0 for request and reply.
```

Comment:
adds routes on a Windows Node for redirecting ClusterIP Service traffic from host network to OVS via antrea-gw0.

Reply:
Updated.